homework/HW_05-01/src/nyc/c4q/m47bell/Domestic.java

```diff
@@ -0,0 +1,13 @@
+package nyc.c4q.m47bell;
+/**
+ * Created by c4q-marbella on 5/1/15.
+ * Access Code 2-1
+ * Marbella Vidals
+ */
+public interface Domestic {
+
+    public void getName();
+
+    public String setName();
```

Comment:
Setters usually take in a parameter, or else you have nothing to set. You should include the parameter in the interface so whoever implements your interface must implement a String name in the method.